chat-widget/yarn.lock

```diff
@@ -5994,6 +5994,13 @@ create-hmac@^1.1.0, create-hmac@^1.1.4, create-hmac@^1.1.7:
     safe-buffer "^5.0.1"
     sha.js "^2.4.8"
 
+cross-fetch@3.1.4:
```

Comment:
Why is yarn.lock modified? Can we revert it?

Reply:
Seems like with the existing package.json file running a yarn install is updating the lockfile.
This is to keep the lockfile updated